xarray-data.ipynb

```diff
@@ -87,14 +87,14 @@
     "import gcsfs\n",
     "\n",
     "fs = gcsfs.GCSFileSystem(project='pangeo-181919', token='anon', access='read_only')\n",
-    "gcsmap = gcsfs.mapping.GCSMap('pangeo-data/newman-met-ensemble',\n",
+    "gcsmap = gcsfs.mapping.GCSMap('pangeo-data/gmet_v1.zarr',\n",
     "                              gcs=fs, check=False, create=False)\n",
     "ds = xr.open_zarr(gcsmap)\n",
     "\n",
     "### Load with intake catalog service\n",
     "# import intake\n",
     "# cat = intake.Catalog('https://raw.githubusercontent.com/pangeo-data/pangeo/master/gce/catalog.yaml')\n",
-    "# ds = cat.newmann_zarr.read_chunked()\n"
+    "# ds = cat.gmet_v1.read_chunked()\n"
```

Comment:
Do you plan on updating the catalog file in-place?
Note that the intake project is being announced in the coming days, so might get more attention and perhaps we should be more purposeful about pangeo's opinion about it. Not urgent, of course.

Reply:
@martindurant - my plan was https://github.com/pangeo-data/pangeo/pull/341

I'm not sure we need to keep `newmann_zarr` around but perhaps for legacy reasons, it would be nice.

Reply:
Keeping the smaller one when pushing out the larger one seems file to me - doesn't waste space too much. 
Nevertheless, thanks for doing this. In the meantime, I have also started the streaming part of the xarray plugin [here](https://github.com/ContinuumIO/intake-xarray/pull/11), which would allow for netcdf to be loaded from a remote Intake server.